spyder/config/base.py

```diff
@@ -360,7 +360,11 @@ def is_py2exe_or_cx_Freeze():
     'zh_CN': u'简体中文',
     'ja': u'日本語',
     'de': u'Deutsch',
-    'pl': u'Polski'
+    'pl': u'Polski',
+    'fa': u'Persian',
+    'hr': u'Croatian',
+    'te': u'Telugu',
+    'uk': u'Ukrainian',
 }
 
 # Disabled languages because their translations are outdated or incomplete
```

Comment:
Please add the keys for the new languages you included above to the list of disabled languages below. That's because the translations for those languages are incomplete right now.